vendor/istio.io/api/mesh/v1alpha1/config.proto

```diff
@@ -42,6 +42,18 @@ enum AuthenticationPolicy {
   INHERIT = 1000;
 }
 
+// LightStepCollector holds configruation for a LightStep Satellite pool.
```

Comment:
I know that these changes will actually get made in istio/api and then vendored in but for ease of prototype testing it seemed easier to just directly modify the vendored files